web_src/less/_user.less

```diff
@@ -169,6 +169,12 @@
   }
 }
 
+.user-badges {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 64px);
+  gap: 2px;
+}
+
```

Comment:
```suggestion

.user-badges img {
  object-fit: contain;
}
```

Reply:
Just built with these two suggestions, and it worked well. Thanks again :)